torch/_dynamo/backends/torchxla.py

```diff
@@ -4,17 +4,21 @@
 from functorch.compile import make_boxed_func
 
 from ..backends.common import aot_autograd
-from ..backends.registry import register_experimental_backend as register_backend
+from ..backends.registry import (
+    register_experimental_backend as register_experimental_backend,
+)
+
+from .registry import register_backend
```

Comment:
```suggestion
from .registry import register_experimental_backend, register_backend
```

Reply:
PyTorch/XLA will throw an error complaining device is not XLA. Let me double check that error message and I can make it more explict.